teku/src/test/java/tech/pegasys/teku/cli/subcommand/ValidatorClientCommandTest.java

```diff
@@ -210,6 +210,23 @@ public void clientExecutorThreadsShouldThrowOverLimit() {
             "--Xvalidator-client-executor-threads must be greater than 0 and less than 5000.");
   }
 
+  @Test
+  public void shouldSetUseDvtSelectionsEndpoint() {
+    final String[] args = {"vc", "--network", "minimal", "--Xdvt-integration-enabled"};
```

Comment:
do we need all of these for this test? i would have thought `--network minimal` wasn't really needed here...